src/configuration.ts

```diff
@@ -122,8 +121,8 @@ export function findConfigurationPath(suppliedConfigFilePath: string | null, inp
         }
     } else {
         // search for tslint.json from input file location
-        let configFilePath = findup(CONFIG_FILENAME, { cwd: inputFilePath, nocase: true });
-        if (configFilePath != null && fs.existsSync(configFilePath)) {
+        let configFilePath = findup(CONFIG_FILENAME, inputFilePath);
+        if (configFilePath !== undefined && fs.existsSync(configFilePath)) {
```

Comment:
is the `fs.existsSync` here necessary? I would expect the file to exist, because we found it in the file system